### DISCUSSION TITLE
MOOSE Team on Vacation!

### DISCUSSION THREAD

**Discussion by aeslaughter** [idaholab/moose#16587] [News]
The MOOSE team and other developers from INL will be on vacation until Jan. 4, 2021. Consequently, our support will be limited during this time. Thanks for all the great questions and contributions in 2020, we look forward to another productive year in 2021. Please take some time to unplug and re-charge your brain meat.

Please feel free to share your success from 2020 with MOOSE here.

**Comment by aeslaughter** [comment]
@lindsayad Is the winner, most commits in the past year!
![Screen Shot 2020-12-24 at 12 48 03 PM](https://user-images.githubusercontent.com/341333/103104755-5d987800-45e6-11eb-9ebd-68213860c8e9.png)

**Reply by lindsayad** [reply]
You win for LOC :-)
